syncall/tw_caldav_utils.py

```diff
@@ -62,9 +62,17 @@ def convert_tw_to_caldav(tw_item: Item) -> Item:
         caldav_item["start"] = tw_item["due"] - timedelta(hours=1)
         caldav_item["due"] = tw_item["due"]
 
+    # Tags
     if "tags" in tw_item.keys():
         caldav_item["categories"] = tw_item["tags"]
 
+    # Project
+    if "project" in tw_item.keys():
+        categories = []
+        categories.extend(tw_item.get("tags", []))
+        categories.append("proj: " + tw_item["project"])
```

Comment:
Hmm, do you need to re-populate the categories with the `tw_item["tags"]` tags ? 
Since that's done in the previous step could you instead skip these three lines (https://github.com/bergercookie/syncall/pull/82/files#diff-c5797fdfe6863a29a849d86701b2dae03a1c11a0c73080337b570d751d26543fR71-R73) and just append the `proj: ...` part?

I'm thinking of code like the following for populating both the category tags and projects:

```
    # Tags and projects
    cats = caldav_item["categories"]  # shorthand
    cats.extend(tw_item.get("tags", []))
    if "project" in tw_item.keys():
        cats.append("proj: " + tw_item["project"])

```

Note that I have not tested this so please try it out :)